bedrock/mozorg/views.py

```diff
@@ -162,6 +162,54 @@ def contribute_signup(request):
     return view_class.as_view()(request)
 
 
+class ContributeTasks(l10n_utils.LangFilesMixin, TemplateView):
+
+    def get_context_data(self, **kwargs):
+        cxt = super(ContributeTasks, self).get_context_data(**kwargs)
+        variation = self.request.GET.get('variation', 'b2')
+        cxt['variation'] = variation
+        return cxt
+
+    template_name = 'mozorg/contribute/contribute-tasks.html'
+
+
+def contribute_tasks(request):
+    return ContributeTasks.as_view()(request)
+
+
+class ContributeTasksSurvey(l10n_utils.LangFilesMixin, FormView):
+    template_name = 'mozorg/contribute/contribute-tasks-survey.html'
+    form_class = ContributeTasksForm
+
+    def get_context_data(self, **kwargs):
+        cxt = super(ContributeTasksSurvey, self).get_context_data(**kwargs)
+        task = self.request.GET.get('task', '')
```

Comment:
possible XSS vector. We should make sure that the `task` value is indeed being escaped in the template. I'd be more comfortable if we were validating the kind of value we're expecting `task` to be.